desktop/test/e2e/specs/login.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-console, import/no-nodejs-modules */
+const { createWriteStream } = require( 'fs' );
+const { mkdir } = require( 'fs/promises' );
+const path = require( 'path' );
+const { _electron: electron } = require( 'playwright' );
+
+// Linux:
+const APP_PATH = path.join( __dirname, '../../../release/linux-unpacked/wpcom' );
+// Mac:
+// const APP_PATH = path.join(
+// 	__dirname,
+// 	'../../../',
+// 	'release',
+// 	'mac',
+// 	'WordPress.com.app',
+// 	'Contents',
+// 	'MacOS',
+// 	'WordPress.com'
+// );
+const CONSOLE_PATH = path.join( __dirname, '../results/console.log' );
+const SCREENSHOT_PATH = path.join( __dirname, '../results/screenshot.png' );
+const HAR_PATH = path.join( __dirname, '../results/network.har' );
+const VIDEO_PATH = path.join( __dirname, '../results/video.webm' );
+
+const timestamp = new Date().toJSON().replace( /:/g, '-' );
+const appLogPath = path.resolve( __dirname, '..', 'results', `app-${ timestamp }.log` );
```

Comment:
This file isn't being created, which implies that the environment variable `WP_DEBUG_LOG` is not being passed through to the app.